packages/knip/src/plugins/prisma/index.ts

```diff
@@ -0,0 +1,37 @@
+import { isDeferResolveEntry, toEntry } from 'src/util/input.js';
+import type { IsPluginEnabled, Plugin, ResolveConfig } from '../../types/config.js';
+import { hasDependency } from '../../util/plugin.js';
+import type { PrismaConfig } from './types.js';
+
+// https://www.prisma.io/docs/orm/prisma-client/setup-and-configuration
+
+const title = 'Prisma';
+
+const enablers = ['prisma', /^@prisma\/.*/];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const config: string[] = ['prisma.config.ts', 'package.json'];
+
+const resolveConfig: ResolveConfig<PrismaConfig> = async (config, options) => {
+  // if config is a package.json file
+  if (config.seed) {
+    const inputs = options.getInputsFromScripts(config.seed);
+    return [...inputs.map(input => (isDeferResolveEntry(input) ? toEntry(input.specifier) : input))];
+  }
```

Comment:
I'm not confident about this code. If I remove this logic and just return the input from `getInputsFromScripts` as-is, then `prisma/seed.ts` was recoginized as `deferResolveEntry` and it was counted in test counter.

Personally, I want this file to be treated as an entry, so I'm transforming it this way. However, I don't fully understand deferResolveEntry, so I'm not sure if this code is correct.

Reply:
Ah, sorry you hit an edge case. Your intentiosns and assumptions were correct, but somewhere in the core of Knip there is a check if `prisma/seed.ts` looks like a package name, and `prisma` also is listed in `package.json`, and that's where it goes awry...

So that has just been fixed in ffa3ca9a4d22b6fe80c7e7f1dee8f38d8d1d2838 so if you rebase from `main` you could just return  `options.getInputsFromScripts` from `resolveConfig` as-is.

Reply:
Thank you for your explanation. I really appreciate the quick fix!